content/features/featuresDeepDive/physics/v2/concepts.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Physics V2 core concepts
+image: 
+description: All the concepts and objects designed for Physics V2 plugins
+keywords: shape, body, material
+further-reading:
+    - title: How To Use Forces
+      url: /features/featuresDeepDive/physics/forces
+video-overview:
+video-content:
+---
+
+# Core concepts
+
+Key concepts of a physics engine include collision shapes, which define the physical shape of an object and determine how it interacts with other objects. Bodies represent physical objects in the simulation, and can have mass, velocity, and other properties. Material properties such as friction, elasticity, and density affect how objects behave when they collide. Other important concepts include constraints, which enforce specific behaviors between objects, and forces, which can be applied to objects to simulate gravity, friction, and other effects.
+
+## Body
+
+A physics body is a virtual object that represents a physical object in a simulation, with properties like mass, position, and velocity.
+
+** center of mass **
+
+## Shape
+
+A physics shape is a virtual representation of the collision geometry of a physics body, used for collision detection and response.
+
+Here are the Shape types support by the V2 Plugin:
+| Enum | Description | Image |
+| --- | --- | --- |
+| SPHERE | Simple sphere | |
+| CAPSULE | A cylinder with a half sphere at top and bottom | |
+| CYLINDER | Cylinder | |
+| BOX | Box | |
+| CONVEX_HULL | A convex hull is the smallest convex shape containing points.| |
+| CONTAINER | Holder of other shapes | |
+| MESH | Mesh used for rendering or a simpler version | |
+| HEIGHTFIELD | A height field mesh is a 2D surface with height data. | |

```

Comment:
We haven't implemented the heightfield yet, so should we take it off the docs?

Reply:
The type is supported in the v2 plugin architecture but not in the plugin that was implemented. We can keep it.